src/quoteservice/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM composer:2.4.1 AS build
+
+WORKDIR /tmp/
+COPY ./src/quoteservice/composer.json .
+
+RUN composer install \
+    --ignore-platform-reqs \
+    --no-interaction \
+    --no-plugins \
+    --no-scripts \
+    --prefer-dist
+
+FROM php:8.1-apache-buster
```

```suggestion
FROM php:8.1-cli
```
Not using apache, so can use this smaller variant